docs/release-1.2/existing-rhdh.md

```diff
@@ -64,121 +91,156 @@ metadata:
 ```bash
 oc get svc -n sonataflow-infra sonataflow-platform-data-index-service -o jsonpath='http://{.metadata.name}.{.metadata.namespace}'
 ```
 
-### app-config ConfigMap
-This ConfigMap used for configuring backstage. Please add/modify to include the following:
-- A static access token (or a different method based on this [doc](https://backstage.io/docs/auth/service-to-service-auth/) to enable the workflows to send notifications to RHDH or to invoke scaffolder actions.
-- Define csp and cors
-
+To include the Notification Plugin:
 ```yaml
-app:
-  backend:
-    auth:
-      externalAccess:
-        - type: static
-          options:
-            token: ${BACKEND_SECRET}
-            subject: orchestrator
-    csp:
-      script-src: ["'self'", "'unsafe-inline'", "'unsafe-eval'"]
-      script-src-elem: ["'self'", "'unsafe-inline'", "'unsafe-eval'"]
-      connect-src: ["'self'", 'http:', 'https:', 'data:']
-    cors:
-      origin: {{ URL to RHDH service or route }}
+      - disabled: false
+        package: "@redhat/plugin-notifications-dynamic@1.2.0"
+        integrity: sha512-1mhUl14v+x0Ta1o8Sp4KBa02izGXHd+wsiCVsDP/th6yWDFJsfSMf/DyMIn1Uhat1rQgVFRUMg8QgrvbgZCR/w==
+        pluginConfig:
+          dynamicPlugins:
+            frontend:
+              redhat.plugin-notifications:
+                dynamicRoutes:
+                  - importName: NotificationsPage
+                    menuItem:
+                      config:
+                        props:
+                          titleCounterEnabled: true
+                          webNotificationsEnabled: false
+                      importName: NotificationsSidebarItem
+                    path: /notifications
+      - disabled: false
+        package: "@redhat/plugin-signals-dynamic@1.2.0"
+        integrity: sha512-5tbZyRob0JDdrI97HXb7JqFIzNho1l7JuIkob66J+ZMAPCit+pjN1CUuPbpcglKyyIzULxq63jMBWONxcqNSXw==
+        pluginConfig:
+          dynamicPlugins:
+            frontend:
+              redhat.plugin-signals: {}
+      - disabled: false
+        package: "@redhat/plugin-notifications-backend-dynamic@1.2.0"
+        integrity: sha512-pCFB/jZIG/Ip1wp67G0ZDJPp63E+aw66TX1rPiuSAbGSn+Mcnl8g+XlHLOMMTz+NPloHwj2/Tp4fSf59w/IOSw==
+      - disabled: false
+        package: "@redhat/plugin-signals-backend-dynamic@1.2.0"
+        integrity: sha512-DIISzxtjeJ4a9mX3TLcuGcavRHbCtQ5b52wHn+9+uENUL2IDbFoqmB4/9BQASaKIUSFkRKLYpc5doIkrnTVyrA==
 ```
 
-To enable the Notifications plugin, edit the same ConfigMaps.
-For the `dynamic-plugins` ConfigMap add:
+Optionally, include the `plugin-notifications-backend-module-email-dynamic` to fan-out notifications as emails.
+The environment variables below need to be provided to the RHDH instance (Or set the values directly in the ConfigMap).
+See more configuration options for the plugin [here](https://github.com/backstage/backstage/blob/master/plugins/notifications-backend-module-email/config.d.ts).
```

Comment:
is `master` branch ok?

Reply:
This was there originally - I'll direct it to @mareklibra as he actually interacted with the referenced file last.

Reply:
I'm okay with master branch. I'm not sure we know how to align the plugin version with its source branch.